native/src/components/PoiDetailRow.tsx

```diff
@@ -26,9 +26,11 @@ type Props = {
 
 const PoiDetailRow = ({ externalUrl, text, accessibilityLabel, icon }: Props): ReactElement => (
   <Row>
-    <Icon source={icon} accessibilityLabel={accessibilityLabel} />
     <StyledPressable onPress={() => openExternalUrl(externalUrl)}>
```

Comment:
Would it be possible to remove one of the `Row` or `StyledPressable` components now and apply all the styling there?